language/de-DE/joomla.ini

```
@@ -1,6 +1,6 @@
-; Joomla! German Translation
+; Joomla. German Translation
 ; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
-; Copyright (C) Translation 2008 - 2019 J!German (www.jgerman.de)
+; Copyright (C) Translation 2008 - 2019 J.German (www.jgerman.de)
```

```suggestion
; Copyright (C) Translation 2008 - 2019 J!German (www.jgerman.de)
```

Eigenname. Bitte nicht anpassen.

---

oops